examples/expositional/frameworks/cortexchat/cortex_chat_quickstart.ipynb

```diff
@@ -0,0 +1,303 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Cortex Chat + TruLens\n",
+    "\n",
+    "This quickstart assumes you already have a Cortex Search Service started, JWT token created and Cortex Chat Private Preview enabled for your account. If you need assistance getting started with Cortex Chat, please contact your Snowflake contact."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set JWT Token and Chat URL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"SNOWFLAKE_JWT\"] = \"...\"\n",
+    "os.environ[\"SNOWFLAKE_CHAT_URL\"] = \".../api/v2/cortex/chat\""
```

Comment:
Thoughts on making this the snowflake account ID and we'll append `/api/v2/cortex/chat` below? Users may not know where to find the Chat API endpoint

Reply:
I'd rather leave this to the Chat team, I'm not sure if that string will be consistent.